packages/grafana-ui/src/components/Flex/Flex.internal.story.tsx

```diff
@@ -0,0 +1,209 @@
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const JustifyContentExamples: StoryFn<typeof Flex> = () => {
+  return (
+    <div>
+      <p>Justify Content space-between</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="space-between" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content space-around </p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="space-around" gap={2}>
+        {Array.from({ length: 3 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content space-evenly</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="space-evenly" gap={2}>
+        {Array.from({ length: 7 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content flex-start</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="flex-start" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content flex-end</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="flex-end" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content center</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const GapExamples: StoryFn<typeof Flex> = () => {
+  return (
+    <div>
+      <p>Gap with spacingToken set to 2 (16px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Gap with spacingToken set to 8 (64px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={8}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Gap with spacingToken set to 10 (80px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={10}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const WrapExamples: StoryFn<typeof Flex> = () => {
+  return (
+    <div>
+      <p>Wrap example with wrap and gap set to spacingToken 8 (64px)</p>
```

Comment:
Lets make these two examples the same thing, as in same gap between items and same number of items, so its easier to compare what the difference between the two are.